admin/connect.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: VALEBLES
+ * Date: 05/02/2019
+ * Time: 16:14
+ */
+
+// connection to database
+require ('../include/config.php');
+
+// we define if the form has been validated
+if(isset($_POST['submit'])) {
+
+    // initialization of error message
+    $errMsg = "";
+
+    // we define the variable with the data filled
+    if (isset($_POST['username']) AND isset($_POST['password'])) {
```

Comment:
remplace AND par &&
on remplace également OR par ||

Reply:
Done